vendor/k8s.io/kubernetes/cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -56,8 +56,7 @@ spec:
           timeoutSeconds: 30
       volumes:
       - name: kubernetes-dashboard-certs
-        secret:
-          secretName: kubernetes-dashboard-certs
+        emptyDir: {}
```

Comment:
does this work? does the dashboard create its own certs at start time?

Reply:
@joelsmith could you respond here?

Reply:
irrelevant for this pick.